### DISCUSSION TITLE
TouchDrag is buggy

### DISCUSSION THREAD

**Discussion by Lissbeat** [ismail9k/vue3-carousel#439] [General]
Hi, I've encountered an issue where touch dragging is buggy on mobile devices. The same problem can be observed in the "Basic" example on this page: [Vue 3 Carousel](https://vue3-carousel.ismail9k.com/examples-fallback.html). It seems like a problem in recent releases, as version 0.3.0 doesn’t have this issue.
![2024-12-05 10 17 07](https://github.com/user-attachments/assets/9c145d11-becc-468b-9f5a-88620a83fc2c)
![2024-12-05 10 37 27](https://github.com/user-attachments/assets/c98dc671-0bff-41fb-85de-20e61a38c0b9)

Is there a way to fix it?

**Comment by ismail9k** [comment]
Hi @Lissbeat,

Thank you for reporting this issue. We are aware of the buggy touch dragging behavior on mobile devices, as you mentioned. This appears to be a regression in recent versions, caused by the use of requestAnimationFrame.

We are actively working on resolving this, and the fix will be included in the next release. You can follow the progress on this issue here: [#438](https://github.com/ismail9k/vue3-carousel/issues/438).

We appreciate your patience and support!

**Comment by ismail9k** [comment]
Fixed in `v0.9.0`
